code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -28,6 +29,13 @@
 				if(check_observer(O))
 					transfer_personality(O)
 			reset_search()
+	else
+		if(silenced)
+			silenced = 0
+			user << "<span class='notice'>You toggle the speaker to 'on', on the [src].</span>"
+		else
+			silenced = 1
+			user << "<span class='notice'>You toggle the speaker to 'off', on the [src].</span>"
 
```

Comment:
This was intentional; I wanted Roboticists to be able to pull in candidates and have them unable to talk, straight away, if they so desired.